### PR TITLE
Ego dump 3

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Just another bog standard Ego dump.
Adds:
Dreaming Current armor
CENSORED armor
Crimson Scar weapons
Various other EGO armor
Adds an easy way to change weapon speed

Resprites:
Paradise Lost
Training Hammer

Nerfs CENSORED to a 100 Weapon (Why was it ever 250 anyways???)
Buffs Wrist Cutter to fit in ego belts (It had Zayin damage on a teth??!?!?!?)
Buffs Gold Rush to 150 from 130. It's a really hard weapon to gauge the power off, but CENSORED is fucking dumb so I guess GR can be too. 

## Why It's Good For The Game
No abnormality should be added without their EGO.
This should fill out some of them.

## Changelog
:cl:
add: Added various Ego
add: Added an easy way for weapons to be sped up
balance:  Censored Nerfed to a 100 weapon
balance:  Wrist Cutter now fits in a belt
balance:  Gold Rush increased to 150 from 130
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 1]
```diff
@@ -342,21 +337,21 @@
 			Looking at the E.G.O for more than 3 seconds will make you sick."
 	special = "Using it in hand will activate its special ability. To perform this attack - click on a distant target."
 	icon_state = "censored"
-	force = 85
+	force = 70	//there's a focus on the ranged atttack here.
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/aleph.dm:340]: There is no focus on ranged... It has high cooldown and subpar damage.

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 2]
```diff
@@ -342,21 +337,21 @@
...
-							JUSTICE_ATTRIBUTE = 90
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
 							)
 
 	var/special_attack = FALSE
-	var/special_damage = 360
+	var/special_damage = 240
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/aleph.dm:354]: And you made it EVEN worse..?

File: code/game/objects/items/ego_weapons/he.dm [hunk 3]
```diff
@@ -33,6 +33,7 @@
 	special = "This weapon has a slower attack speed."
 	icon_state = "fury"
 	force = 45
+	attack_speed = 0.5
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/he.dm:36]: Why did you make it faster?

File: code/game/objects/items/ego_weapons/aleph.dm (hunk 1 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/aleph.dm:340]: I had to balance it for 100 PE somehow.
It being 80 with a free 360 damage attack is a little too much to be on par with Mimicry and Smile

File: code/game/objects/items/ego_weapons/he.dm (hunk 3 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/he.dm:36]: This was a mistake.

File: code/modules/clothing/suits/ego_gear/waw.dm [hunk 4]
```diff
@@ -140,3 +140,27 @@
...
+
+/obj/item/clothing/suit/armor/ego_gear/throne
+	name = "false throne"
+	desc = "And here I sit upon a throne of lies."
+	icon_state = "throne"
+	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 0, BLACK_DAMAGE = 40, PALE_DAMAGE = 70)
+	attribute_requirements = list(
+							INSIGHT_ATTRIBUTE = 60)
+
+/obj/item/clothing/suit/armor/ego_gear/throne
```
Comment (dsmith328) [inline code/modules/clothing/suits/ego_gear/waw.dm:160]: These have the same path

**Comment by dsmith328** [comment]
ego_weapons.dmi has two extra images that have invalid names

**Pull request merged**